src/main/utils/sleep.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+export function sleep(ms: number): Promise<void> {
+    return new Promise(r => setTimeout(r, ms));
+}
```

Comment:
this would likely be better in shared/utils

Reply:
how did i miss that